operators/postgresql/5.1.0/manifests/postgresoperator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,381 @@
+      Set your [pod anti-affinity][k8s-anti-affinity], node affinity, Pod tolerations and more rules to customize your deployment topology!
+    - **Full Customizability**: Crunchy PostgreSQL for Kubernetes makes it easy to get your own PostgreSQL-as-a-Service up and running
+      and fully customize your deployments, including:
+        - Choose the resources for your Postgres cluster: [container resources and storage size][resize-cluster]. [Resize at any time][resize-cluster] with minimal disruption.
+        - Use your own container image repository, including support `imagePullSecrets` and private repositories
+        - [Customize your PostgreSQL configuration][customize-cluster]
+
+    and much more!
+
+    [backups]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/backups/
+    [clone]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/disaster-recovery/#clone-a-postgres-cluster
+    [customize-cluster]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/customize-cluster/
+    [disaster-recovery]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/disaster-recovery/
+    [high-availability]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/high-availability/
+    [monitoring]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/monitoring/
+    [pool]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/connection-pooling/
+    [provisioning]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/create-cluster/
+    [resize-cluster]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/resize-cluster/
+    [tls]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/customize-cluster/#customize-tls
+
+    [k8s-anti-affinity]: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity
+    [k8s-nodes]: https://kubernetes.io/docs/concepts/architecture/nodes/
+
+    [pgBackRest]: https://www.pgbackrest.org
+    [pgBouncer]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/connection-pooling/
+    [pgMonitor]: https://github.com/CrunchyData/pgmonitor
+
+
+    ## Post-Installation
+
+    ### Tutorial
+
+    Want to [learn more about the PostgreSQL Operator][tutorial]? Browse through the [tutorial][] to learn more about what you can do!
+
+    [tutorial]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial
+  version: '5.1.0'
+  links:
+    - name: Crunchy Data
+      url: https://www.crunchydata.com/
+    - name: Documentation
+      url: https://access.crunchydata.com/documentation/postgres-operator/v5/
+  maintainers:
+    - name: Crunchy Data
+      email: info@crunchydata.com
+  minKubeVersion: 1.20.0
```

Comment:
@tjmoore4 you are setting k8s min version to `1.20` (ocp `4.7`) and in annotations you are setting it to `v4.6`. Please fix one of these

Reply:
Can you put `minKubeVersion: 1.19.0` and this would work

Reply:
@mvalarh since the EOL for Kubernetes is distinct from OpenShift, we treat these values differently/distinctly.  Specifically, since Kubernetes v1.19 is no longer actively being maintained, we do not support that version of Kubernetes.  However, with OpenShift providing extended update support for v4.6 (i.e Kube 1.19), we do support OpenShift v4.6.  This also aligns with how we test and validate our operator, along with the supported versions we define in our documentation, etc.

Additionally, we have three other v5.1.0 bundles/PR's already merged in with that same setting:
- https://github.com/k8s-operatorhub/community-operators/pull/1109
- https://github.com/redhat-openshift-ecosystem/certified-operators/pull/751
- https://github.com/redhat-openshift-ecosystem/redhat-marketplace-operators/pull/165

Are the requirements different for bundles in this repository?  We would like to keep these details properly aligned and set accurately across all of our OLM installers.

Reply:
:+1: Thanks for the explanation, I've pushed that update.